tpl/data/data.go

```diff
@@ -97,9 +106,9 @@ func (ns *Namespace) GetCSV(sep string, urlParts ...interface{}) (d [][]string,
 // GetJSON expects one or n-parts of a URL to a resource which can either be a local or a remote one.
 // If you provide multiple parts they will be joined together to the final URL.
 // GetJSON returns nil or parsed JSON to use in a short code.
-func (ns *Namespace) GetJSON(urlParts ...interface{}) (interface{}, error) {
+func (ns *Namespace) GetJSON(args ...interface{}) (interface{}, error) {
 	var v interface{}
-	url := joinURL(urlParts)
+	url := joinURL(args)
```

Comment:
All of this looks mostly great. I have a few comments_

If I read correctly, any headers will be joined to the string here. What I would suggest is to create a separate function that splits the URL parts of the slice and the header part, and add a small test for that.